### PR TITLE
Syntheses

### PR DESCRIPTION
**Pull request by vincentfretin**
modification of the v1 frontend to edit synthesis is still in progress

### CODE REVIEW

File: assembl/graphql/schema.py [hunk 1]
```diff
@@ -718,6 +771,9 @@ def resolve_title(self, args, context, info):
     def resolve_synthesis_title(self, args, context, info):
         return resolve_langstring(self.synthesis_title, args.get('lang'))
 
+    def resolve_phase_identifier(self, args, context, info):
+        return "thread"
```
Comment (n0izn0iz) [inline assembl/graphql/schema.py]: I'm tring to do the real implem of this

File: assembl/static2/css/components/ideaSynthesis.scss [hunk 2]
```diff
@@ -95,3 +102,22 @@ $small-margin: 5px;
     display: block;
   }
 }
+
+.idea-synthesis.idea-synthesis-level-2{
```
Comment (amensouissi) [inline assembl/static2/css/components/ideaSynthesis.scss]: The css styles associated to the ideaSynthesis component are not finished...

File: assembl/static2/js/app/pages/synthesis.jsx [hunk 3]
```diff
@@ -29,20 +30,20 @@ export class DumbSynthesis extends React.Component<void, SynthesisProps, void> {
...
-              {...rootIdea}
-              index={index + 1}
-              parents={[]}
-              subIdeas={getChildren(rootIdea, children)}
-              slug={routeParams.slug}
-            />
-          );
-        })}
-
+        <Col md={10} xsOffset={2}>
```
Comment (amensouissi) [inline assembl/static2/js/app/pages/synthesis.jsx]: Please use mdOffset here... and complete the css style

File: assembl/static2/js/app/components/synthesis/IdeaSynthesis.jsx [hunk 4]
```diff
@@ -1,52 +1,20 @@
...
-import { getPartialTree, getChildren } from '../../utils/tree';
+import { getSentimentsCount, createDoughnutElements } from '../debate/common/announcement';
 
-const createTooltip = (sentiment, count) => {
+const SynthesisBody = ({ level, value, Addon }) => {
   return (
-    <Tooltip id={`${sentiment.camelType}Tooltip`} className="no-arrow-tooltip">
-      {count} <Translate value={`debate.${sentiment.camelType}`} />
-    </Tooltip>
+    <div className="synthesis-body" style={{ columnCount: level > 2 ? 2 : 'auto' }}>
```
Comment (amensouissi) [inline assembl/static2/js/app/components/synthesis/IdeaSynthesis.jsx]: columnCount must be 'auto' for small media

File: assembl/static2/js/app/utils/section.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+// @flow
```
Comment (paolina92) [inline assembl/static2/js/app/utils/section.js]: Same here, maybe use `levelSection`

File: assembl/static2/css/components/section.scss [hunk 6]
```diff
@@ -0,0 +1,13 @@
+.section-title{
```
Comment (paolina92) [inline assembl/static2/css/components/section.scss]: Maybe `levelSection.scss` is more verbose

**Pull request merged**